client/src/pages/Home.js

```diff
@@ -39,7 +39,7 @@ class BaseHome extends Page {
 	}
 
 	constructor(props) {
-		super(props, DEFAULT_PAGE_PROPS, CLEAR_SEARCH_PROPS)
+		super(props, DEFAULT_PAGE_PROPS, DEFAULT_SEARCH_PROPS)
```

Comment:
Change this to simply `super(props)` everywhere (they're the defaults after all) and ditch the imports.